tests/unit/requests/test_download.py

```diff
@@ -98,6 +98,39 @@ def test__write_to_stream_with_hash_check_success(self):
             chunk_size=download_mod._SINGLE_GET_CHUNK_SIZE,
             decode_unicode=False)
 
+    def test__write_to_stream_with_raw_response(self):
+        stream = io.BytesIO()
+        download = download_mod.Download(EXAMPLE_URL, stream=stream)
+
+        chunk1 = b'\x1f\x8b\x08\x08?"\xeaY\x02\xfftmpBazYVY\x00KLJNIMK\xcf' \
+                 b'\xc8\xcc\xca\xce\xc9\xcd\xcb/(,*.)-+\xaf\xa8'
+        chunk2 = b'\xac20426153\xb7\xb0\xe4J\x1cU4\xaahT\xd1\xa8\xa2QE\xa3' \
+                 b'\x8aF\x15\x8d*'
+        chunk3 = b'\x1aU\x04S\x04\x00\x04P\xf8\xea@\t\x00\x00'
+        header_value = u'crc32c=qmNCyg==,md5=KHRs/+ZSrc/FuuR4qz/PZQ=='
+        headers = {download_mod._HASH_HEADER: header_value,
+                   'content-encoding': 'gzip'}
+
+        response = _mock_response(chunks=[chunk1, chunk2, chunk3],
+                                  headers=headers)
+
+        value = iter([chunk1, chunk2, chunk3])
+        mock_patch = mock.patch(u'google.resumable_media.requests.download.'
+                                u'Download._read_chunk_raw_response',
+                                return_value=value)
+        with mock_patch as mock_method:
+            ret_val = download._write_to_stream(response)
+
+            assert ret_val is None
+
+        assert stream.getvalue() == chunk1 + chunk2 + chunk3
+
+        # Check mocks.
```

Comment:
This comment doesn't have much meaning. I wonder why they have had it there before.

Reply:
Done